src/fcmcts/fcmcts.jl

```diff
@@ -292,6 +292,7 @@ POMDPLinter.@POMDP_require insert_node!(tree::FCMCTSTree, planner::FCMCTSPlanner
     S = eltype(SV)
 
     # TODO: Review IQ and IN
+    # Should this be ::S or ::SV? We can have global state that's not a vector.
```

Comment:
Wait, can we? Because we were doing `enumerate(S)` before when collecting the actions. Is `enumerate` possible on anything other than a vector?

Reply:
Yeah, this is something I am not sure about. Technically there can be some "global" information that is not provided to agents but then it becomes partially observed?

Reply:
I think `SV`, but see my main review comment.